content/automate/ManageIQ/System/CommonMethods/QuotaMethods.class/__methods__/requested.rb

```diff
@@ -146,6 +146,12 @@ def vmdb_object(model, id)
   $evm.vmdb(model, id.to_i) if model && id
 end
 
+def get_disk_size(disk_name)
+  mydisk = $evm.vmdb('disk').find_by(:filename => disk_name)
+  ManageIQ::Automate::System::CommonMethods::Utils::LogObject.log_and_raise("ERROR - Disk_name not found for Reconfiguration", @handle) if mydisk.nil?
```

Comment:
@billfitzgerald0120 
Do we need a spec for this?
Also we are missing the method instance yaml with the list of embedded methods